### DISCUSSION TITLE
Completion menu is hiding preview window

### DISCUSSION THREAD

**Discussion by uwla** [neoclide/coc.nvim#3279] [Q&A] [answered]
For some reason the completion menu is hiding the preview window. How to put the preview window somewhere else?

![screenshot-2021-08-09-18-16-03](https://user-images.githubusercontent.com/47862859/128775715-68ac0d90-2aad-473d-8150-ce411174fa26.png)

**Comment by fannheyward** [comment, marked as answer]
Should be fixed https://github.com/neoclide/coc.nvim/commit/fe35d5fe2bba69b6fcbb0555e4a58f5187dcef80, upgrade your coc.

**Reply by uwla** [reply]
After upgrading Coc I'm not  getting a good suggestion menu (it shows less suggestion, and less relevant ones), neither a preview window : c
